internal/services/network/private_link_service_resource_test.go

```diff
@@ -254,7 +254,7 @@ resource "azurerm_subnet" "test" {
   virtual_network_name = azurerm_virtual_network.test.name
   address_prefixes     = ["10.5.4.0/24"]
 
-  enforce_private_link_service_network_policies = true
+  private_link_service_network_policies_enabled = false
```

Comment:
Many of these are getting set to `false` when they were `true` before, could you explain why?